static/app/utils/performanceForSentry/index.tsx

```diff
@@ -41,7 +41,7 @@ export function onRenderCallback(
   actualDuration: number
 ) {
   try {
-    const transaction: Transaction | undefined = getPerformanceTransaction();
```

Comment:
This was throwing an error on getsentry for some reason?

Reply:
because `IdleTransaction` is removed from the sdk? 

I see in the code:
```
export function getPerformanceTransaction(): IdleTransaction | Transaction | undefined {
 ... snip ...
}

... 

const transaction: Transaction | undefined = getPerformanceTransaction();
```